tests/testthat/test_rpart_2.R

```diff
@@ -50,6 +52,8 @@ test_that("exp_rpart(binary) evaluate training and test", {
   model_df <- flight %>% dplyr::mutate(is_delayed = as.logical(`is delayed`)) %>%
                 exp_rpart(is_delayed, `DIS TANCE`, `DEP DELAY`, test_rate = 0.3, binary_classification_threshold=0.5)
 
+  ret <-  model_df %>% rf_partial_dependence()
```

Comment:
Does this cover following case?
- data including NA
- data with NA
- data including Inf/-Inf
- data with Inf/-Inf

Reply:
Those are handled before building models.